tsparser/src/parser/resources/apis/encoding.rs

```diff
@@ -291,7 +291,7 @@ fn describe_req(
                 None,
             ));
         } else {
-            anyhow::bail!("missing request schema");
+            anyhow::bail!("request schema must be defined when having path parameters");
```

Comment:
I think we should also ensure all path params are present in the schema, unless that's very hard

Reply:
We do that, but thats a different error, looks like this:
```
error: path parameter "hello" not found in request schema
  --> /home/fredr/projects/encore-apps/teeeest/repro/repro.ts:29:27
   |
29 |   export const withParams = api(
   |  ___________________________^
30 | |   { expose: true, method: "POST", path: "/some/:hello" },
31 | |   async (req: Req) => {},
32 | | );
   | |_^
```

Reply:
Ah great!